modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/model/SearchKeyword.java

```diff
@@ -104,7 +104,7 @@ public JSONObject toJSONObject() {
 		).put(
 			"searchVolume", getSearchVolume()
 		).put(
-			"traffic", getTraffic()
+			"traffic", (int)getTraffic()
```

Comment:
Better use Math.toIntExact(Long), here is a nice blog about why you should use it: https://dzone.com/articles/exact-conversion-of-long-to-int-in-java

Reply:
Good catch, pushing..